src/chat/chat_panel.rs

```diff
@@ -185,74 +183,7 @@ live_design! {
         }
     }
 
-    PromptButton = <CircleButton> {
```

Comment:
Most of this has been extracted to `prompt_input.rs`.

Reply:
Although, logic controlling most of this remains here. Probably should be abstracted inside.